src/ophys_etl/modules/segmentation/utils/roi_utils.py

```diff
@@ -321,3 +323,57 @@ def roi_list_from_file(file_path: pathlib.Path) -> List[OphysROI]:
             roi = OphysROI.from_schema_dict(roi_data)
             output_list.append(roi)
     return output_list
+
+
+def select_contiguous_region(
+        seed_pt: Tuple[int],
+        input_mask: np.ndarray) -> np.ndarray:
+    """
+    Select only the contiguous region of an ROI mask that contains
+    a specified seed_pt
+
+    Parameters
+    ----------
+    seed_pt: Tuple[int]
+        The (row, col) coordinate of the seed point that
+        must be contained in the returned mask
+
+    input_mask: np.ndarray
+        A mask of booleans
+
+   Returns
+   -------
+   contiguous_mask: np.ndarray
+       A mask of booleans corresponding to the contiguous
+       block of True pixels in input_mask that contains seed_pt
+    """
+    if not input_mask[seed_pt[0], seed_pt[1]]:
```

Comment:
fwiw, this will fail with an `IndexError` right here is seed_pt entries are outside of the input mask size.
Shouldn't happen, but, up to you if you want a more informative exception there.

Reply:
I added a custom message to the IndexError. Thanks for noticing this.